package.json

```diff
@@ -0,0 +1,11 @@
+{
```

Comment:
Do you also want to update the bin/deploy script to run the yarn install command

Reply:
yes, that's a good idea. And it would also need to run encore. But maybe we should wait until it'll done? So we optimize it.